src/cljs/ui/new_analysis/discrete_rates.cljs

```diff
@@ -44,49 +43,53 @@
              :or   {burn-in 0.1}}
             @bayes-factor
             controls-disabled? (or (seq? @field-errors)
-                                   (not log-file-name))]
+                                   (not log-file-name))
+            ;; weird JS behaviour, where it will parse floats with full precision
+            burn-in (round burn-in 2)
```

Comment:
I don't get the precision removing thing? can't it just have all the precision and do a format for displaying on the slider?

Reply:
It makes a round-trip when setting is sent to the server - mutations response is handled by the graphql event handler and put into app db, such that when starting the analysis the db value could differ from the displayed - confusing for the user.

Reply:
but why do we need to remove precision? that is what I don't understand

Reply:
The slider increments by 0.1, all the extra digits are just some artifacts of JS handling numbers (e.g. 0.2 returned in a server response becomes 0.2000000000000000019)